### DISCUSSION TITLE
[2022/04/28] OpenFunction Community Meeting

### DISCUSSION THREAD

**Discussion by benjaminhuo** [OpenFunction/community#7] [General]
Agenda:
- Python functions-framework &  sync/async function update, @kehuili
- Multiple routes https://github.com/OpenFunction/functions-framework/issues/17， @lizzzcai 
- KEDA-HTTP addon intro & demo, @tpiperatgod 
- Parameter validation proposal https://github.com/OpenFunction/OpenFunction/issues/274, @tpiperatgod 
- Java Quarkus runtime discussion https://github.com/quarkiverse/quarkus-dapr, @benjaminhuo
- Support build functions with the Dockerfile method, @benjaminhuo
- CNCF sandbox onboarding process https://github.com/cncf/toc/issues/829, @feynman @benjaminhuo
- Publish an official announcement for onboarding the CNCF sandbox project @Feynman Zhou @yangchuansheng  Yang

https://docs.google.com/document/d/1bh5-kVPegjNlIjjq_e37mS3ZhyXWhmmUaysFgeI9_-o

**Comment by liZe** [comment]
I think that you’re pinging the wrong @liZe 😁.

**Reply by tpiperatgod** [reply]
fixed 🙂
